eng/testing/tests.singlefile.targets

```diff
@@ -24,11 +24,12 @@
   <PropertyGroup Condition="'$(TestNativeAot)' == 'true'">
     <IlcToolsPath>$(CoreCLRILCompilerDir)</IlcToolsPath>
     <IlcToolsPath Condition="'$(TargetArchitecture)' != '$(BuildArchitecture)'">$(CoreCLRCrossILCompilerDir)</IlcToolsPath>
```

Comment:
This condition should be updated to ensure that our NativeAOT libraries tests link against the target rootfs instead of the host image binaries.